vdr/vdr.go

```diff
@@ -163,3 +171,50 @@ func (r *VDR) Deactivate(DID did.DID, current hash.SHA256Hash) {
 	logging.Log().Debugf("Deactivating DID Document: %s", DID)
 	panic("implement me")
 }
+
+func (r *VDR) resolveControllers(doc did.Document) ([]did.Document, error) {
+	return r._resolveControllers([]did.Document{doc})
+}
+
+func (r *VDR) _resolveControllers(input []did.Document) ([]did.Document, error) {
```

Comment:
I don't like stuff in Golang starting with an underscore. Feels a bit like "this is even more private than the one without the udnerscore, but Go doesn't support it". I'd say collaps the two and just call the one with the array. You can even make it vararg (`...`) so it works for both (recursive and initial call).